varcode/transcript_mutation_effects.py

```diff
@@ -313,7 +315,7 @@ def get_class(effect):
             return _MultipleSubstitution
     return effect.__class__
 
-variant_effect_priority_list = [
+transcript_effect_priority_list = [
```

Comment:
this get_class function and the fake _MultipleSubstitution class seem hacky to me. Could this logic and also the places where the top ranked variant is selected be instead done with a sort_variant_effect_by_priority(variants) function? All of this logic could then at least be internal to that function. Or, maybe better: just make a function called variant_effect_priority(variant) -> int that returns a priority number for that variant, and can then be used as a key function in sorted()